plugins/lookup/bitwarden.py

```diff
@@ -29,7 +29,7 @@
           - Field to retrieve, for example V(name) or V(id).
           - If set to V(id), only zero or one element can be returned.
             Use the Jinja C(first) filter to get the only list element.
-          - When O(collection_id) is set, this field can be undefined to retrieve the whole collection records.
+          - When this field is undefined, retrieves the whole records (filtered by O(collection_id) and O(organization_id) if set).
         type: str
         default: name
```

Comment:
Since there's a default, you cannot leave it undefined. Or what exactly do you mean by "is undefined"?

Reply:
I just took the language used by the previous line.

But thanks for asking, I actually read it as `term`, but this is the docs for `search`.
Therefore, I'd suggest:

`If set to `None` or `''`, or if the O(_terms) is None, records are not filtered by fields.`

Reply:
> I just took the language used by the previous line.

Ah, I see. Yeah, that one isn't that much better then ;)

> But thanks for asking, I actually read it as `term`, but this is the docs for `search`. Therefore, I'd suggest:
> 
> > If set to `None` or `''`, or if the O(_terms) is None, records are not filtered by fields.

`_terms` cannot be `None`, it can only be an empty list, or a list containing `None`. So how about:

> > If set to V(None) or V(''), or if O(_terms) is empty, records are not filtered by fields.